### PR TITLE
Feature/top bar

### PR DESCRIPTION
**Pull request by ardier16**

### CODE REVIEW

File: src/vue/common/TabsPanel.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="tabs-panel">
+    <div class="tabs-panel__main">
+      <slot />
```
Comment (max-nes) [inline src/vue/common/TabsPanel.vue]: the component actually doesn’t accept anything that looks like "tabs", also maybe it makes sense to rename the component to better describe the role it plays. "tabs-component" sounds not such global as it have to be, something describing it role "position on top panel" would be great.

Comment (ardier16) [reply src/vue/common/TabsPanel.vue]: Resolved, updated

**Comment by ardier16** [comment]
Example of using TopBar: https://gist.github.com/ardier16/35d2d576675671a535a1717728a92d82

File: src/vue/common/TopBar.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      0
+      -1 * $content-side-paddings-sm
+      4 * $point
+      -1 * $content-side-paddings-sm;
+    flex-direction: column;
+    align-items: start;
+
+    .top-bar__extra {
+      margin-top: 2 * $point;
+      margin-left: 0;
```
Comment (max-nes) [inline src/vue/common/TopBar.vue]: `flex-wrap: wrap`

File: src/vue/common/TopBar.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      <slot name="main" />
+    </div>
+    <div class="top-bar__extra">
+      <slot name="extra" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```
Comment (max-nes) [inline src/vue/common/TopBar.vue]: let’s add a short usage manual. you can use `Drawer.vue` as a reference

Comment (ardier16) [reply src/vue/common/TopBar.vue]: Resolved, added

File: src/vue/common/TopBar.vue (hunk 2 above)
Comment (ardier16) [reply src/vue/common/TopBar.vue]: Resolved, updated

**Pull request merged**